src/SignalR.Orleans/OrleansHubLifetimeManager.cs

```diff
@@ -213,9 +202,10 @@ public void Dispose()
         }
     }
 
+    [Serializable]

```

Comment:
I don't think we need that on our types anymore.

Reply:
I m not sure if its need it or not, I added it to be safe :) 
If you're sure i will remove it

Reply:
You should be safe on dropping that. Lets see the test results 💃